Library/Formula/racer.rb

```diff
@@ -0,0 +1,41 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+
```

Comment:
How about adding:

```
head "https://github.com/phildawes/racer.git"
```

here? 1.0.0 is 100+ commits behind head and head is the recommended way of installing it.